ideas/121-swarm-compensation.md

```diff
@@ -0,0 +1,72 @@
+## Preamble
+
+    Idea: 121-swarm-compensation
+    Title: Swarm Compensation
+    Status: In Progress
+    Created: 2018-04-02
+    Related: `120-swarm-process`
+
+## Summary
+Ultimately we would like Status to become a public good, owned, developed and ran by it's very own users. In order to achieve that we need to open up our processes for contributions from "the outside" and establish ways for users to propose & incentivise projects they see as important.
+
+Swarms are our current attempt at enabling this. They are ephemeral teams forming to achieve a specified goal and disbanding once achieved or failure becomes apparent. Their defined lifetime and -cycle encourage constant re-evaluation &. evolution.
+
+Compensating contributors for their work is an important component of this and where `120-swarm-process` focuses on establishing a baseline of processes that are open to everyone this idea will focus on experimentally devising compensation mechanisms. 
+
+## Swarm Participants
+
+- Lead Contributor: @martinklepsch
+- Contributor ('payment'): ??? 
+- Contributor: @oskarth
+- Evaluation ('compensation criteria'): Voting mechanism (Polly survey)
+
+## Product Overview
+
+This swarm will define **four** experimental compensation mechanisms and recruit upcoming swarms to participate in this experiment.
+
+The compensation mechanisms should be judged according to the following criteria:
```

Comment:
Good stuff

Reply:
I wonder if we need to explicitly state that the purpose of this is to find ways where compensation can be fairly distributed to core and non-core contributors, therefore blurring the lines between those who are "employees" and those who are not?

This also means changing the payment structure of employees from pure salary/hourly based, to a different structure that can scale.

Reply:
Yeah, might be worth adding.

> This also means changing the payment structure of employees from pure salary/hourly based, to a different structure that can scale.

I'd like to challenge the notion that salaries can't scale 🙂